config.go

```diff
@@ -490,6 +490,8 @@ type ModuleConfig interface {
 	// WithStartFunctions configures the functions to call after the module is
 	// instantiated. Defaults to "_start".
 	//
+	// Clearing the default is supported, via `WithStartFunctions()`.
```

Comment:
here's a reminder you can clear this. we clear it when we explicitly time `_start`, but most frameworks that use WASI or FFI won't clear it as it makes init require more logic and error handling, hence this config.